topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -215,59 +238,84 @@ In order to convert the chromosome names we have therefore two things to do:
 
 > ### {% icon hands_on %} Hands-on: Adjust chromosome names
 >
-> 1. **Replace Text** {% icon tool %}: Run **Replace Text in a specific column**  with the following settings:
->     - As **File to process** our peak file `GSE37268_mof3.out.hpeak.txt`
->     - **in column**: `1`
+> 1. **Replace Text** {% icon tool %}: Run **Replace Text in a specific column** with 
+>     - **File to process** to our peak file `GSE37268_mof3.out.hpeak.txt`
+>     - **in column**: `Column:1`
 >     - **Find pattern**: `[0-9]+` (this will look for numerical digits)
 >     - **Replace with**: `chr&` (`&` is a placeholder for the find result)
-> 2. Click **Execute**
-> 3. Let's rerun the tool for the final changes two times. You can shortcut this by pressing the **rerun icon** in the history, but don't forget to adjust the settings:
->    - As **File to process** use the output from the last run, e.g. something like `Replace Text on data ...`
->    - **in column**: `1`
+> 3. **Replace Text** {% icon tool %}: Let's rerun the tool with
+>    - **File to process** to the output from the last run, e.g. something like `Replace Text on data ...`
+>    - **in column**: `Column:1`
 >    - **Find pattern**: `chr20`
 >    - **Replace with**: `chrX`
-> 4. Rerun this tool accordingly for chromosome Y.
-> 5. Wait for the jobs to finish and inspect the latest file through the **eye icon**. Have we been successful?
+>
+>    > ### {% icon tip %} Tip: Rerunning a tool
+>    >
+>    > * Press the **rerun icon** in the history
+>    {: .tip}
+>
+> 4. **Replace Text** {% icon tool %}: Rerun this tool accordingly for chromosome Y
+> 5. Inspect the latest file through the **eye icon**
+>
+>    Have we been successful?
 >
 {: .hands_on}
 
-We have quite some files now and should take care that we don't loose track. Let's rename our latest result to something more handy, e.g. `Peak regions`.
+We have quite some files now and should take care that we don't loose track. Let's rename our latest result to something more handy, *e.g.* `Peak regions`.
```

Comment:
Italic for `e.g.` and `i.e.` are usually not recommended, see https://english.stackexchange.com/questions/3911/correct-spelling-italicization-of-e-g-i-e